__tests__/app.test.js

```diff
@@ -27,4 +27,26 @@ describe('/api/topics', () => {
             })
         })
     });
+});
+
+
+
+describe.only('/api/articles/:article_id', () => {
+    test('should get an article object specified by the endpoint', () => {
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then((result) => {
+            const { article } = result.body
+            expect(article[0]).toMatchObject({
```

Comment:
Minor but an important detail. `article` is an array containing only a single object - it should instead just be that object